src/scripts/settings/userSettings.ts

```diff
@@ -278,7 +317,9 @@ export class UserSettings {
      * @param {string|undefined} val - Language.
      * @return {string} Language.
      */
-    customCss(val) {
+    customCss(): string | null;
+    customCss(val: string): void;
+    customCss(val?: string): string | null | void {
```

Comment:
SonarCloud seems to prefer a type alias to be used instead of three unions for `string | null | void`.

Reply:
Done in [1503113](https://github.com/jellyfin/jellyfin-web/pull/5065/commits/150311324335d353d89ac100c651905afb6dd1d6)